tests/func/test_utils.py

```diff
@@ -69,3 +73,19 @@ def test_boxify(self):
         )
 
         assert expected == utils.boxify("message")
+
+
+@pytest.mark.skipif(os.name == "nt", reason="Not supported for Windows.")
+def test_makedirs_permissions():
+    dir_mode = 0o755
+    intermediate_dir = "тестовая-директория"
+    test_dir = os.path.join(intermediate_dir, "data")
```

Comment:
You are creating this directory in the projects root, which is not nice. Please use `tmpdir` fixture or something like that.

Reply:
good point, simplified using tmpdir